scipy-1.0/paper.tex

```diff
@@ -281,6 +281,26 @@ \subsection*{Data structures}
 \input{ckdtree}
 \textbf{Sparse matrices}
 
+\texttt{scipy.sparse} offers seven sparse matrix data structures. 
+The most important ones are CSR and CSC; these offer fast row or column 
+access and fast matrix-vector multiplication, and are used heavily in 
+scikit-learn.
+
+Over the last three years our sparse matrix handling internals have
+been rewritten and performance has been improved. Iterating over 
+and slicing of CSC and CSR matrices is now faster by up to 35\%,
+and the CSR to COO matrix format conversion is now several times faster.
+COO matrices are now generally more memory efficient. Importantly,
```

Comment:
Remind me why COO is more memory efficient now?

Reply:
I thought I picked that up from parsing the release notes, but it may have been misintrepreted or just quite minor--I'll just remove the statement if it is not something substantial that happened.

Reply:
Are you sure you're not mixing it up with pydata/sparse#158 and pydata/sparse#164? Those are kind-of related, but they aren't in `scipy.sparse`